network-api/networkapi/wagtailpages/forms.py

```diff
@@ -24,3 +24,16 @@ def clean_parent(self):
             if self.instance == parent:
                 self.add_error("parent", "A category cannot be a parent of itself.")
         return parent
+
+
+# Max number validation for blog page topics. We are using a custom form as if
+# we do this validation in the BlogPage models clean() method, it will return the
+# error as a 500, and not as an error in the admin panel.
```

Comment:
I am still confused by this comment. 

How about this: 
```suggestion
# Max number validation for blog page topics. We are using a custom form to avoid
# an issue where the `ValidationError` raised in the page's `clean` method is
# not caught and turned into a form error by Wagtail.
```